HOW_TO_MARK.md

```diff
@@ -24,7 +24,7 @@ https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1-Solution
 
 ### 1.2) Using the Solutions Repo
 
-In these repositories you will find solutions to each weeks coursework. These solutions are example answers and will not be the exact solution that students give. You should use it to inform your feedback of the coursework.
+In these repositories you will find solutions to each week's coursework. These solutions are example answers and will not be the exact solution that students give. You should use it to inform your feedback of the coursework.
```

Comment:
PS these are template files so you might prefer to pull in the updated version from here:

https://github.com/CodeYourFuture/CYF-Coursework-Template/blob/main/HOW_TO_MARK.md

and make changes to that template file to fix everywhere when you see errors like this